WalletWasabi.Gui/Global.cs

```diff
@@ -156,9 +156,9 @@ public async Task InitializeNoWalletAsync(TerminateService terminateService)
 						await TorManager.StartAsync(ensureRunning: true).ConfigureAwait(false);
 					}
 
-					var httpClient = (WalletWasabi.Tor.Http.TorHttpClient)Synchronizer.HttpClientFactory.NewHttpClient(() => Config.GetFallbackBackendUri(), isolateStream: false);
+					var httpClient = (WalletWasabi.Tor.Http.TorHttpClient)Synchronizer.HttpClientFactory.NewHttpClient(() => null!, isolateStream: false);
```

Comment:
This does not seem to be good, but I don't know a better option.

Reply:
This is something I mentioned here: https://github.com/zkSNACKs/WalletWasabi/pull/5079#discussion_r564482276 

I think that the relative URI should be optional. Then it would be just: `HttpClientFactory.NewHttpClient(isolateStream: false);` and it would be more in line with how `HttpClient` in .NET works.